pkg/cloudprovider/aws/apis/v1alpha1/provider_defaults.go

```diff
@@ -26,7 +26,6 @@ import (
 func (c *Constraints) Default(ctx context.Context) {
 	c.defaultArchitecture()
 	c.defaultCapacityTypes()
-	c.defaultAMIFamily()
```

Comment:
So this won't show up on the provisionerSpec anymore right since we're in-memory defaulting now?

Reply:
Yes, that is correct.